package-lock.json

```diff
@@ -473,25 +473,25 @@
       }
     },
     "node_modules/@babel/compat-data": {
-      "version": "7.23.5",
-      "resolved": "https://registry.npmjs.org/@babel/compat-data/-/compat-data-7.23.5.tgz",
```

Comment:
These changes seem unrelated, are the packages' existing versions incompatible with Node.js 20.x?